db/migrate/20200819184941_add_user_to_sessions.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToSessions < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :sessions, :user
```

Comment:
Can you verify this creates a bigint column? We have other PRs where I'm not sure.

Reply:
Yes, just verified it in my local DB

Reply:
Same as the `id` column, `int8`